src/swap/SwapScreen.tsx

```diff
@@ -759,6 +801,16 @@ export function SwapScreen({ route }: Props) {
           text={t('swapScreen.transactionDetails.networkFeeInfoDismissButton')}
         />
       </BottomSheet>
+      <BottomSheetInLineNotification
+        showNotification={!!selectingNoUsdPriceToken}
+        severity={Severity.Warning}
+        title={t('swapScreen.noUsdPriceWarning.title')}
+        description={t('swapScreen.noUsdPriceWarning.description')}
+        ctaLabel={t('swapScreen.noUsdPriceWarning.ctaConfirm')}
```

Comment:
should we reverse the CTAs to match the design?

Reply:
yes done! i think the design was changed, or i was crazy